google/cloud/bigtable/examples/table_admin_snippets.cc

```diff
@@ -259,8 +288,14 @@ void GenerateConsistencyToken(google::cloud::bigtable::TableAdmin admin,
 
   //! [generate consistency token]
   [](google::cloud::bigtable::TableAdmin admin, std::string table_id) {
-    std::string token = admin.GenerateConsistencyToken(table_id);
-    std::cout << std::endl << "generated token is : " << token << std::endl;
+    auto token = admin.GenerateConsistencyToken(table_id);
+    if (!token) {
+      std::cerr << "GenerateConsistencyToken failed: " << token.status()
+                << std::endl;
+      return;
+    }
+    std::cout << std::endl
```

Comment:
I realize this leading newline was here before your change, but is it needed? Do you know why this is here? I ask because I'm not seeing this in the other examples.